ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -71,6 +71,7 @@
   private static final int INPUT_TYPE_KEYBOARD_NUMBERED =
       InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_FLAG_DECIMAL |
```

Comment:
should TYPE_CLASS_NUMBER be removed here?

Reply:
@chirag04 Yes, I already cleanup the code just now